modules/ROOT/pages/advanced-usage.adoc

```diff
@@ -4,18 +4,18 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: agent, mule, esb, servers, monitor, notifications, external systems, third party, get status, metrics
 
-image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-Usually, all you need to do with the Runtime Manager Agent is install it and configure it according to  xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring the Runtime Manager Agent].
+Usually, all you need to do with Runtime Manager agent is install it and configure it according to  xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update Runtime Manager Agent].
 
-If you are curious about the underlying architecture and the advanced configuration settings for the Runtime Manager Agent, explore these topics:
+If you are curious about the underlying architecture and the advanced configuration settings for Runtime Manager agent, explore these topics:
```

Comment:
Global question: So if the proper name isn't Runtime Manager Agent, then shouldn't we have an article? 
i.e. "Runtime Manager Agent" or "the Runtime Manager agent"?